content/docs/guides/crossguard/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: CrossGuard
+
+menu:
+    userguides:
+        parent: guides
+        identifier: crossguard
+---
+
+{{% crossguard-preview %}}
+
+CrossGuard is Pulumi's new Policy as Code offering. CrossGuard allows you to empower your developers to provision their own infrastructure while providing them with guardrails. Using Policy as Code, you can write flexible business or security policies.
```

Comment:
Tone: "your developers" only makes sense if the audience is explicitly someone in the management chain. Perhaps instead: "empowers you to set guardrails to enforce compliance for resources ... so developers within an organization can ..."?